docs/docs/workflows/create.md

```diff
@@ -70,13 +176,27 @@ Create the `constellation-state.yaml` using the output from the Terraform state
 
 ```bash
 CONSTELL_IP=$(terraform output ip)
-CONSTELL_INIT_SECRET=$(terraform output initSecret | jq -r | tr -d '\n' | base64)
-touch constellation-state.yaml
-yq eval '.version ="v1"' --inplace constellation-state.yaml
-yq eval '.infrastructure.initSecret ="$CONSTELL_INIT_SECRET"' --inplace constellation-state.yaml
-yq eval '.infrastructure.clusterEndpoint ="$CONSTELL_IP"' --inplace constellation-state.yaml
+CONSTELL_INIT_SECRET=$(terraform output initSecret | yq -r | tr -d '\n' | base64)
```

Comment:
I need to leave a comment to request changes again. So this is a reference to https://github.com/edgelesssys/constellation/pull/2458/files#r1362159049.